tests/units/Migration.php

```diff
@@ -69,14 +69,6 @@ function () {
       }
    }
 
-   public function testConstructor() {
```

Comment:
Maybe we can test output is empty; since this is what is expected? (same for the update test)